fuzz/action.yml

```diff
@@ -62,11 +70,17 @@ runs:
     - name: Run fuzzer
       shell: bash
       run: |
-        cd archipelago && uv run python fuzz.py -j 4 -g ${APWORLD_NAME} -r ${RUNS} -n ${YAMLS_PER_RUN}
+        if [[ ${{ inputs.dump-ignored }} == 'true' ]]; then
```

Comment:
I don't like this whole thing.

For dump-ignored you can do `${DUMP_IGNORED:+--dump-ignored}` and for the meta you can do `${META:+-m $META}`